Resources/Prototypes/_Impstation/CosmicCult/tempdoc/thecosmiccult.md

```diff
@@ -0,0 +1,321 @@
+
```

Comment:
This entire file is in a folder called tempdoc, so presumably you didn't want to commit this?